rero_ils/modules/patrons/views.py

```diff
@@ -43,6 +48,50 @@
 )
 
 
+_PID_REGEX = re.compile(r'NOT\s+pid:(\w+)')
```

Comment:
should be written as `_PID_REGEX = re.compile(r'NOT\s+pid:\s*(\w+)\s*')` to avoid problem if `pid` is prefixed or suffixed with spaces